invite/views.py

```diff
@@ -157,12 +159,15 @@ def log_in_user(request):
 @login_required(redirect_field_name=None,
                 login_url=reverse_lazy('invite:login'))
 def index(request):
+    after_date = date.today() - timedelta(weeks=12)
```

Comment:
Do you think the timeframe of displayed invitations should be a setting?

Reply:
For some reason that thought did not occur to me. But it makes sense, I'll add that as a setting.